### ISSUE TITLE
Incorrect rendering

### ISSUE THREAD

**Issue by samyakbhuta** [kartikm/fonts-rekha#3]
See following image taken from Gujarati Wikisource Logo.
![bvrsdmocuaavlud](https://f.cloud.github.com/assets/738746/1351956/6f1849d4-372f-11e3-9d42-78c1b2469029.jpg)

Ref : https://twitter.com/0x1f1f/status/385995447002599424

**Comment by kartikm** [comment]
Difficult to fix until #1 is fixed :(

**Comment by samyakbhuta** [comment]
Yes, I understand. Just wanted to be documented here before it goes back
into oblivion a.k.a. twitter timeline !
